pkg/controller/postgressnapshot/postgressnapshot_controller_test.go

```diff
@@ -1,210 +0,0 @@
-package postgressnapshot
```

Comment:
Deleted this file and the redissnapshot_controller_test.go also because they were only unit testing a create method that doesn't exist in the controller anymore. There are unit tests for the new providers that check their create methods